routes/trading.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express')
+const router = express.Router()
+const authenticate = require('../middlewares/authenticate')
+const { trade } = require('../controllers/tradingController')
+const { newTrade } = require('../middlewares/validators/trading')
+
+/**
+ * @swagger
+ * /trade:
```

Comment:
Can you please change this to `/trade/:username` ?

Reply:
We have a new endpoint - https://github.com/Real-Dev-Squad/website-backend/pull/222/commits/5bae0772542bd974fcefb7f2c60b6519b88f8da8